src/database.c

```diff
@@ -0,0 +1,215 @@
+/*
+ * file: database.h
+ * authors: 
+        Carter S. Levinson <carter.levinson@pitt.edu>
+        Qizhe Wang <qiw68@pitt.edu>
+ *         
+ * status: working
+ */
+
+#include "database.h"
+#include "ntapfuse_ops.h"
+
+#include <stdio.h>
+#include <string.h>
+#include <time.h>
+#include <stdlib.h>
+
+#include <unistd.h>
+#include <errno.h>
+#include <limits.h>
+
+#include <sqlite3.h>
+
+#define TIME_MAX 80
+#define BUF_MAX 600
+#define QUOTA 1000000
+
+/* global pointer to database connection object */
+sqlite3 *DB;
+
+/*TODO: add commented documentation here for each function sig*/
+/*TODO: figure out meaning of null parameters in SQL exec */
+
+int open_db() {
```

Comment:
Start { on the new line for functions. For other code, blocks place it on the same line - to improve readability.

Reply:
fixed, thanks